dashboard/database.py

```diff
@@ -177,37 +170,32 @@ def get_all_tags(_conn: Connection) -> list:
 
 
 @st.cache_data(ttl="1hr")
-def get_sales_by_country(_conn: Connection, n: int = 5) -> pd.DataFrame:
+def get_sales_by_country(_conn: Connection):
```

Comment:
I think you mistakenly removed the expected return type hint here.

Reply:
yes, added back in now